internal/jimm/access_test.go

```diff
@@ -834,9 +834,13 @@ func TestGetGroupByID(t *testing.T) {
 	c.Assert(err, qt.IsNil)
 	c.Assert(groupEntry.UUID, qt.Not(qt.Equals), "")
 
-	gotGroup, err := j.GetGroupByID(ctx, u, groupEntry.UUID)
+	gotGroupUuid, err := j.GetGroupByUUID(ctx, u, groupEntry.UUID)
 	c.Assert(err, qt.IsNil)
-	c.Assert(gotGroup, qt.DeepEquals, groupEntry)
+	c.Assert(gotGroupUuid, qt.DeepEquals, groupEntry)
+
+	gotGroupName, err := j.GetGroupByName(ctx, u, groupEntry.Name)
```

Comment:
add also some tests when the group is not found

Reply:
Already covered by tests for the API client, but just to be sure I'll add this here too.